t/05-messages/01-errors.t

```diff
@@ -228,6 +228,19 @@ throws-like { Blob.splice }, X::Multi::NoMatch,
         'longer method names are suggested also';
 }
 
+# https://github.com/rakudo/rakudo/issues/1758
+{
+    throws-like q| class GH1758_1 { submethod x { }; }; class B is GH1758_1 {}; B.new._ |,
+        X::Method::NotFound,
+        :message{ !.contains: "Did you mean 'x'" },
+        'Ancestor submethods should not be typo-suggested';
+
+    throws-like q| class GH1758_2 { submethod x { };}; GH1758_2.new._ |,
+        X::Method::NotFound,
+        message => /"Did you mean 'x'"/,
```

Comment:
For consistency with the previous test, shouldn't this be `:message{ .contains: "Did you mean 'x'" },`?

Reply:
I use that style because the rest of the file did that for positive matching, but I can change it.

Reply:
Not that big a deal. If it's consistent with something that's fine. All the tests (across NQP, Rakudo, and roast) aren't very consistent on the whole.